### ISSUE TITLE
When using `patchDocument()` the custom numbering flow doesn't work.

### ISSUE THREAD

**Issue by martijn10kb** [dolanmiu/docx#2088]
When using `patchDocument()` the custom numbering flow doesn't work. Or if it's possible where would we insert the INumberingOptions object?
When trying to open a docx that has been generated that has custom numbering it fails to open.

**Comment by yarosalvpp** [comment]
Same thing, trying to figure out how to make numbering paragraphs patching document

**Comment by lcampanis** [comment]
This is required, not possible to do at the moment...

**Comment by valentindoering** [comment]
I am even unable to insert a simple bullet point using patchDocument. 
When the document is opened in Libre Office or Google Docs, the bullet point appears as plain text instead of as a bullet point.

```js
const updatedDocBuffer = await patchDocument(docBuffer, {
  patches: {
      placeholder: {
          type: PatchType.DOCUMENT,
          children: [ new Paragraph({ text: "Bullet point content", bullet: { level: 0 }} ],
      }
})
```

Are you experiencing similar issues? @lcampanis @yarosalvpp @martijn10kb

**Comment by martijnenco** [comment]
The 'workaround' we did was to just inject custom xml into the styling.xml of the word document..
If you do this you will need to parse the xml, find out if your style is already present if not then inject it, save/find the style-list-id and then in your document.xml make sure all the list have that style-list-id..
Hope that make sense to you.. ..At the time I couldn't find a libruary capable of this.. But if it exists now, I would go with that instead of our work around

**Comment by martijnenco** [comment]
Yeah at this point you will need to do a search and replace all with the numId, then you gott it (i think) havent looked at our code for a while now, so its from memory for me
